examples/chef/common/clusters/temperature-control/static-supported-temperature-levels.h

```diff
@@ -21,32 +21,62 @@
 #include <app/clusters/temperature-control-server/supported-temperature-levels-manager.h>
 #include <app/util/config.h>
 
+namespace chef {
+namespace Configuration {
+namespace TemperatureControl {
+
+/**
+ * @brief Endpoint to temperature levels mapping. The endpoint must have a temperature control cluster.
+ * Represents a pair of endpoints and temperature levels supported by that endpoint.
+ */
+struct EndpointPair
+{
+    /**
+     * @brief An endpoint having temperature control cluster.
+     */
```

Comment:
```suggestion
     /// An endpoint having temperature control cluster.
```

Nitpicky thoughout: consider more vertical-compact text to reduce signal to noise ratio (non-blocking PR on this ... just thought of this when seeing the extra whitespace and the noisy `@brief` parts)